### DISCUSSION TITLE
Geodan

### DISCUSSION THREAD

**Discussion by saxelsson** [OlivierZal/com.melcloud#282] [Ideas]
Hi, I really like your app, I’m using it to my geothermal heatpump Geodan even though it’s not supposed to support it. Everything works well but I’m missing values from two temp sensors. It is the temp on the liquid from and to the ground. In the Geodan system they are named TH32 and TH34. Do you have any plan of adding these sensors?

**Comment by OlivierZal** [comment]
Hi @saxelsson,

Thanks for your feedback!

if I found the values on your MELCloud API, I can add these values without any problem.

For this I need a guest access to your API, but I'll come back to you when I'm back from holidays :-)

Kind regards

**Comment by OlivierZal** [comment]
@saxelsson in order to ease the discussion, would you be ok to join https://slack.athom.com/ ?

**Reply by saxelsson** [reply]
Sorry for the delay. Now I’ve joined Slack. I’ll DM you.

**Comment by OlivierZal** [comment]
After investigation: not exposed by the API.
